az-cvm-vtpm/src/vtpm/mod.rs

```diff
@@ -126,6 +128,24 @@ impl Quote {
         let nonce = attest.extra_data().to_vec();
         Ok(nonce)
     }
+
+    pub fn verify_pcrs(&self) -> Result<bool, QuoteError> {
```

Comment:
The other verification functions in the crate do not return `Result<bool, _>` but `Result<(), VerifyError>`. For consistency we should keep that.

It's also tricky for api users because they might call `quote.verify_pcrs()?` and assume the verification passed, while the result of the verification is discarded. It's hard to enforce this via compiler warnings because the Result has indeed been consumed, it's just the wrapped value that is ignored.

Reply:
done